core/src/transaction_builder.rs

```diff
@@ -231,22 +296,21 @@ pub fn create_kickoff_utxo_tx(
     }
 }
 
-#[tracing::instrument(ret(level = tracing::Level::TRACE))]
 pub fn create_slash_or_take_tx(
     deposit_outpoint: OutPoint,
     kickoff_utxo: UTXO,
-    operator_xonly_pk: &XOnlyPublicKey,
+    operator_xonly_pk: XOnlyPublicKey,
     operator_idx: usize,
-    nofn_xonly_pk: &XOnlyPublicKey,
+    nofn_xonly_pk: XOnlyPublicKey,
     network: bitcoin::Network,
     user_takes_after: u32,
     operator_takes_after: u32,
     bridge_amount_sats: u64,
 ) -> TxHandler {
     // First recreate the move_tx and move_txid. We can give dummy values for some of the parameters since we are only interested in txid.
-    let move_tx_handler = create_move_tx(
+    let move_tx_handler = create_move_tx_handler(
```

Comment:
Why not create_move_tx here

Reply:
fbb190ff5f05995246feed3479c33a528b0bc7ca